pipeline/outputs/kafka.md

```diff
@@ -114,3 +115,28 @@ specific avro schema.
     rdkafka.log_level 7
     rdkafka.metadata.broker.list 192.168.1.3:9092
 ```
+
+#### Kafka Configuration File with Raw format
+
+This example fluent-bit config creates dummy records with the _payloadkey_ and
+_msgkey_ keys. The value of the _msgkey_ will be used as the Kafka message key
+and the value of the _payloadkey_ as payload.
+
+
+```text
+[INPUT]
+    Name dummy
+    Tag  dummy.data
```

Comment:
```suggestion
    Name example
    Tag  example.data
```

Reply:
"dummy" is not an inclusive term, and can be seen as a pejorative, so we avoid those words where possible.

Reply:
@lockewritesdocs this is referring to the dummy input so the `name` has to be that: https://docs.fluentbit.io/manual/pipeline/inputs/dummy